routes/index.js

```diff
@@ -1,5 +1,9 @@
 const router = require("express").Router();
 
 router.use("/api", require("./api"));
+// return for call to /api
+router.get("/api", (req, res) => {
```

Comment:
Why did you edit this route? By updating this route, you have cut off the rest of the routes and controllers. Please attend to it by reversing the changes.